### PR TITLE
Dịch "kaggle-house-price_vn" - Phần 3

### LINKED ISSUE #1018: Dịch "kaggle-house-price_vn" - Phần 3
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_multilayer-perceptrons/kaggle-house-price_vn.md#L169-L274

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by quangnhat185**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Trả lời số issue sau dấu `#` (không có dấu cách): Close #1018 

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
    * Các phần in nghiêng, in đậm.
    * Tiêu đề (số lượng dấu `#` đầu dòng).
    * Bảng biểu, chú thích cho bảng (dòng phía trên mỗi bảng bắt đầu bằng dấu `:`).
    * Dịch các từ trong hình vẽ nếu cần. Các dòng có hình có dạng: `caption`.
    * Dịch các chú thích hình vẽ (thay các cụm `*dịch chú thích ảnh phía trên*` bằng bản dịch tiếng Việt).
    * Không dịch các phần code (nằm giữa hai dấu `````).
    * Copy các công thức toán từ bản gốc (các đoạn có `$`).
    * Giữ các dòng gán nhãn (bắt đầu với `:label:`).
    * Không tự thêm bớt các dòng trắng.
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn.

* [x] Tất cả chữ trong các hình vẽ đã được dịch. Nếu chưa, vui lòng xem thêm **[hướng dẫn hoặc yêu cầu dịch hình](https://github.com/aivivn/d2l-vn/blob/master/img/README.md)**.

* [x] Điền tên của bạn vào mục "Những người thực hiện" ở cuối file.

* [ ] **Sau khi được approve**, điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Với Reviewer

1. Chỉ review nếu Pull Request này đã sẵn sàng cho việc review.

2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

3. **Bạn có thể đề xuất thay đổi cách dịch mỗi dòng trực tiếp như trong hình dưới đây:**
img

### CODE REVIEW

File: chapter_multilayer-perceptrons/kaggle-house-price_vn.md [hunk 1]
```diff
@@ -185,16 +185,23 @@ You can partition the training set to create a validation set, but you will only
...
 -->
 
-*dịch đoạn phía trên*
-
+Lưu ý rằng dữ liệu của cuộc thi được tách thành tập huấn luyện và tập kiểm tra.
+Mỗi tập dữ liệu bao gồm giá trị tài sản của ngôi nhà và các thuộc tính liên quan bao gồm loại đường phố, năm xây dựng, loại ngói, tình trạng tầng hầm, v.v.
+Các đặc trưng được biểu diễn bởi nhiều kiểu dữ liệu.
+Ví dụ, năm xây dựng được biểu diễn bởi số nguyên, loại ngói là các lớp đặc trưng riêng biệt, các đặc trưng khác thì được biểu diễn bởi số thực dấu phẩy động (_floating point number_).
+Và đây là khi ta đối mặt với vấn đề thực tiễn: ở một vài mẫu, nhiều dữ liệu bị thiếu và được chú thích đơn giản là 'na'.
+Giá của mỗi căn nhà chỉ được đưa vào tập huấn luyện (sau cùng thì đây vẫn là một cuộc thi).
```
Comment (lkhphuc) [inline chapter_multilayer-perceptrons/kaggle-house-price_vn.md]:
```suggestion
Giá của mỗi căn nhà chỉ được cung cấp trong tập huấn luyện (sau cùng thì đây vẫn là một cuộc thi).
```

File: chapter_multilayer-perceptrons/kaggle-house-price_vn.md [hunk 2]
```diff
@@ -185,16 +185,23 @@ You can partition the training set to create a validation set, but you will only
...
 so you will want to make sure that you have `pandas` installed before proceeding further. 
 Fortunately, if you are reading in Jupyter, we can install pandas without even leaving the notebook.
 -->
 
-*dịch đoạn phía trên*
+Chúng ta sẽ đọc và xử lý dữ liệu với `pandas`, một [công cụ phân tích dữ liệu hiệu quả](http://pandas.pydata.org/pandas-docs/stable/), vì vậy hãy đảm bảo rằng bạn đã cài đặt `pandas` trước khi tiếp tục.
+Một điều may mắn là, nếu bạn đang sử dụng Jupyter, bạn có thể cài đặt pandas mà không cần thoát khỏi notebook. 
 
 ```{.python .input  n=3}
 # If pandas is not installed, please uncomment the following line:
```
Comment (lkhphuc) [inline chapter_multilayer-perceptrons/kaggle-house-price_vn.md]: Những comment như thế này có nên dịch không nhỉ?

File: chapter_multilayer-perceptrons/kaggle-house-price_vn.md [hunk 3]
```diff
@@ -230,7 +237,7 @@ DATA_HUB['kaggle_house_test'] = (
 To load the two csv files containing training and test data respectively we use Pandas.
 -->
 
-*dịch đoạn phía trên*
+Để nạp lần lượt hai file csv tập huấn luyện và tập kiểm tra, ta sử dụng Pandas.
```
Comment (lkhphuc) [inline chapter_multilayer-perceptrons/kaggle-house-price_vn.md]:
```suggestion
Ta sử dụng Pandas để nạp lần lượt hai tập tin csv chứa dữ liệu huấn luyện và kiểm tra.
```

File: chapter_multilayer-perceptrons/kaggle-house-price_vn.md (hunk 2 above)
Comment (duythanhvn) [reply chapter_multilayer-perceptrons/kaggle-house-price_vn.md:207]: Ông giáo đang nghĩ lại sao, mình những ngày đầu dự án đè ra dịch hết, anh em phản đối điều này 😄

File: chapter_multilayer-perceptrons/kaggle-house-price_vn.md [hunk 4]
```diff
@@ -688,7 +699,7 @@ với dấu `@` ở đầu. Ví dụ: @aivivn.
 *
 
 <!-- Phần 3 -->
-*
+* Nguyễn Lê Quang Nhật
```
Comment (quangnhat185) [inline chapter_multilayer-perceptrons/kaggle-house-price_vn.md:702]:
```suggestion
* Nguyễn Lê Quang Nhật
* Lê Khắc Hồng Phúc
```
